ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -10,7 +10,8 @@ STS=${STS:-true}
 HOSTED_CP=${HOSTED_CP:-false}
 COMPUTE_MACHINE_TYPE=${COMPUTE_MACHINE_TYPE:-"m5.xlarge"}
 OPENSHIFT_VERSION=${OPENSHIFT_VERSION:-}
-CHANNEL_GROUP=${CHANNEL_GROUP}
+TARGET_VERSION="${VERSION}"
+IFS=' ' read -r -a CHANNEL_GROUPS <<< "${CHANNEL_GROUPS_OVERRIDE:-nightly candidate stable}"
```

Comment:
Where would the value for `CHANNEL_GROUPS_OVERRIDE` be coming from?  Also, this could lead to the same situation as before, where a `channel` value is given but then nothing is found.

I think what would need to happen is that if a `channel` value was given that the other channels would then still need to be searched if the desired `channel` doesn't contain the version.